pages/02.Setup/01.Getting Started/docs.en.md

```diff
@@ -109,23 +109,23 @@ The next step after creating the database is to set up your administrator accoun
 
 ![admin user](admin-user.png)
 
-Enter a username, password, first and last name and an email address, and the account will be created for you.
+Enter a username, password, first and last name, and an email address, and the account will be created for you.
 
 ### Email configuration
 
 The next step allows you to configure the way in which email is sent from your Mautic instance.
 
 ![email config](email-config.png)
 
-If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials and authentication mode for the systems you wish to use.
+If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send an email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials, and authentication mode for the systems you wish to use.
```

Comment:
Could you possibly fix these links to use the [link macro markup syntax](https://docs.mautic.org/en/home/contributing#absolute-links)?